test/integration/targets/iosxr_l3_interfaces/tests/cli/_populate_config.yaml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Populate Config
+  cli_config:
+    config: "{{ lines }}"
+  vars:
+    lines: |
+      interface GigabitEthernet 0/0/0/2
+      ip address 192.168.1.1 255.255.255.0
```

Comment:
All IP addresses in integration tests should be from the documentation range in order to avoid collision with actual IPs in the test infra.